Reference/caves.json

```diff
@@ -2593,6 +2602,26 @@
       "z": -448
     }
   },
+  {
+    "type": "cave",
+    "name": "Sundial Sanctum",
+    "specialInfo": "§eCave Complex",
+    "description": "§7Enter The Sanctum at §f[1062, 73, -1726]§7 if you dare §d󐀄§bSailers",
+    "length": "long",
+    "lengthInfo": "15m",
+    "difficulty": "hard",
+    "requirements": 35,
+    "rewards": [
+      "10000 XP",
+      "1 §bUnidentified Solar",
```

Comment:
Should there really be formatting in the reward type name?

Reply:
We have formatting in this file in many other places. We basically only use the name and coordinates from the file itself. WIth mapdata, we will be deprecating this file anyways. If you check `combat-mapfeatures`, they are fully correct.